Cookbook/Technical-Documents/ToolsAndServices.md

```diff
@@ -7,17 +7,18 @@
-| GitHub | Repos for the app, the bots, the playbook... | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/248) to get added to the org. Make sure your Babylon email is added to your account. |
+| Zeplin | Screen Designs | Ask the support engineer in the iOS team channel on Slack for access to the project(s). Depending on your tribe you might need to use Figma |
+| [Figma](https://www.figma.com/) | Screen Designs | Log in with Google as this should redirect to the Babylon Okta sign in. Depending on your tribe you might need to use Zeplin |
+| GitHub | Repos for the app, the bots, the playbook... | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/248) to get added to the org. Make sure you add your GitHub username in the support ticket, and that your Babylon email is added to your account |
 | [Lokalise](https://lokalise.co/projects) | Manage translations used by the app | Use iOS team vault on 1Password to gain access |
 | Confluence Spaces (iOS, your Tribe, ...) | Documentation for various processes | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) – You can also ask your PM or Tribe Lead which Spaces you need to be given access to. (Since we are moving away from Confluence this might not be necessary) |
 | Google Team Drives | Access shared documents for which you need multiple people to contribute | Login using your Microsoft account which is used for email – You can also ask your PM / Tribe Lead which Drives you need to be given access to. You should be given access to at least `iOS` and `Consumer Apps Tribe`|
 | [zoom.us](https://zoom.us) | Video Conferences | Create account with BabylonHealth email-Id. You will have access to basic plan (you can join invites without an account but will need one to create conferences)|
 | VPN | Remote Access to our intranet and internal domains | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) |
 | [CircleCI](https://circleci.com/gh/babylonhealth) | See CI jobs and logs | Use your GitHub login |
-| [Pull Panda](https://pullreminders.com) | Stats on PRs and reminders for when you've been asked for a review | Use your GitHub login |
-| [App Center](http://appcenter.ms/apps) | see OTA builds sent internally to QA | Ask support engineer in the iOS team channel on Slack to invite you to the org |
+| [App Center](http://appcenter.ms/apps) | See OTA builds sent internally to QA | Log in using company sign in. Once you have access to App Center, ask the support engineers in the iOS team channel on Slack to invite you to the org or use the shared credentials to invite yourself if you already have access to 1Password. For more information, there's [documentation](https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Technical-Documents/AppCenter.md) focused on this.
```

Comment:
@MichalTKwiecien SSO is working for App Centre, isn't it?

Reply:
From Slack messages, SSO kinda works, some users can't see build, some can. Let's keep this for now